flake.nix

```diff
@@ -374,15 +388,42 @@
           '';
         };
 
-        git-hooks-nix = {
-          baseUrl = "https://github.com/cachix/git-hooks.nix/blob/master";
+        pkgs-by-name-for-flake-parts = {
+          title = "pkgs-by-name-for-flake-parts";
+          baseUrl = "https://github.com/drupol/pkgs-by-name-for-flake-parts/blob/main";
           intro = ''
-            Configure pre-commit hooks.
+            [pkgs-by-name-for-flake-parts](https://github.com/drupol/pkgs-by-name-for-flake-parts) is a flake-parts
+            that can autoload Nix packages under a particular directory.
 
-            Generates a configuration for [pre-commit](https://pre-commit.com),
-            provides a script to activate it, and adds a [check](flake-parts.html#opt-perSystem.checks).
+            The structure is identical to the [Nixpkgs one](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md).
```

Comment:
This isn't accurate; it behaves somewhat like https://nixos.org/manual/nixpkgs/stable/index.html#function-library-lib.filesystem.packagesFromDirectoryRecursive

It fulfills a similar purpose, but the details are quite different.

Just referring to aforemented function doesn't cut it, I'm afraid, because of the issue with directories _below_ `pkgsDirectory` producing an `attrsOf package` instead of a `package`. That's a type error.